pykg2tbl/service.py

```diff
@@ -75,13 +75,18 @@ def __init__(self, *files):
         for f in files:
             log.debug(f"loading graph from file {f}")
             graph_to_add = g.parse(f)
-            self.graph = graph_to_add if self.graph is None else self.graph + graph_to_add
```

Comment:
what would happen if this line was just `self.graph += graph_to_add`

Reply:
be better written. I am changing all these classes in another branch, so maybe leave that to a different PR